lib/components/narrative/line-itin/line-itinerary.js

```diff
@@ -77,18 +75,13 @@ export default class LineItinerary extends NarrativeItinerary {
           timeOptions={timeOptions}
         />
 
-        {user &&
-          <span className='pull-right'><LinkButton to='/savetrip'>Save this option</LinkButton></span>
-        }
-
```

Comment:
Actually, I know this goes against what I previously stated, but we should keep the new SaveTrip button component here `<SaveTripButton />`. This was decided in conversation with Ritesh earlier last week.

Reply:
Reinstated in https://github.com/opentripplanner/otp-react-redux/pull/266/commits/80b8a912a73dce09a55078b0a243478e14dc783c.

Reply:
https://github.com/opentripplanner/otp-react-redux/pull/266/commits/dec353af2400b589ee2998747ea450d76d468e4d uses `<SaveTripButton />` in `LineItinerary`.